app/server.rb

```diff
@@ -0,0 +1,41 @@
+require 'hobbit'
```

Comment:
really glad you've done this. Using hobbit over sinatra is a great way to up you web dsl's